nemo/collections/llm/api.py

```diff
@@ -513,18 +513,23 @@ def evaluate(
     from nemo.collections.llm import evaluation
 
     # Get tokenizer from nemo ckpt. This works only with NeMo 2.0 ckpt.
-    tokenizer = io.load_context(nemo_checkpoint_path + '/context', subpath="model").tokenizer
+    tokenizer = io.load_context(nemo_checkpoint_path + "/context", subpath="model.tokenizer")
     # Wait for rest service to be ready before starting evaluation
     evaluation.wait_for_rest_service(rest_url=f"{url}/v1/health")
     # Create an object of the NeMoFWLM which is passed as a model to evaluator.simple_evaluate
     model = evaluation.NeMoFWLMEval(
         model_name, url, tokenizer, max_tokens_to_generate, temperature, top_p, top_k, add_bos
     )
     results = evaluator.simple_evaluate(
-        model=model, tasks=eval_task, limit=limit, num_fewshot=num_fewshot, bootstrap_iters=bootstrap_iters
+        model=model,
+        tasks=eval_task,
+        limit=limit,
+        num_fewshot=num_fewshot,
+        bootstrap_iters=bootstrap_iters,
+        batch_size=8,
```

Comment:
is adding a batch size intentional for this change too? or a leftover from testing?

Reply:
Good catch, removed

Reply:
That's a good find. I overlooked it. Actually `batch_size` is not supported yet with our code (as we are using a custom model to interface with lm-eval-harness) so even if this is passed it does nothing. So good to remove it to avoid confusion and I will add it with the batching support.

Reply:
Oops sorry hadn't seen the latest comment.